neurokit2/ecg/ecg_intervalrelated.py

```diff
@@ -25,7 +25,9 @@ def ecg_intervalrelated(data, sampling_rate=1000):
     DataFrame
         A dataframe containing the analyzed ECG features. The analyzed features consist of the following:
 
-        * ``ECG_Rate_Mean``: the mean heart rate.
+        .. codebookadd::
+            ECG_Rate_Mean|The mean heart rate.
+
         * ``ECG_HRV``: the different heart rate variability metrices.
```

Comment:
Yes this was done intentionally, because I was thinking about what should be added to the codebook and I wasn't sure, if adding ECG_HRV which is basically an entry holding entries that are already accounted for the the HRV section. However, I can also see an argument for adding these "meta keys" to the codebook too. What do you think?

Reply:
I think I would at least have a way to filter out "meta-entries" within the code book generation process, rather than formatting them differently in the docstring, to avoid inconsistent formatting in the documentation. It could also be helpful to have these variables saved somewhere if we wanted to programmatically validate the output of the codebook later on.

Reply:
We can probably solve this using a third column which could be empty for normal variables and for "meta" keys, we could add the reference to the docs location which would give the link to the documentation that shows you what the key contains. How does that sound?

e.g., for ECG_HRV, it would look like this in codebookadd: `ECG_HRV|the different heart rate variability metrics|<path to function page>` where the 3rd column adds a :doc: in rst which links to the page. All other variables in the directive would not have to be changed